Automattic/jetpack
# Fix editor connections list for simple sites

There was some regression in #36976 that resulted in simple site users being asked to connect their WPCOM accounts.

## Proposed changes:

* Fix social connections list in editor not displayed for simple sites

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this to sandbox
* Point your simple site to your sandbox
* Goto post editor
* Open Jetpack sidebar
* Confirm that you see social connections instead of the notice to connect your WPCOM acccount.
